packages/cli/src/operations.ts

```diff
@@ -29,3 +43,38 @@ function isOpenAPI3(document: unknown): document is OpenAPIObject {
 function isPostmanCollection(document: unknown): document is CollectionDefinition {
   return Array.isArray(get(document, 'item')) && get(document, 'info.name');
 }
+
+// TODO: remove, need https://github.com/stoplightio/json/pull/90
+export const decycle = (obj: unknown, replacer?: (value: any) => any) => {
+  const objs = new WeakMap<object, string>();
```

Comment:
Let's make it in fp-ts style.

Reply:
This function is going to be replaced, with the json package one.